### ISSUE TITLE
[WH] Onslaught 4 pcs Proc does not increase Critical Hit Rate

### ISSUE THREAD

**Issue by CrimsonFenris** [WarEmu/WarBugs#18372]
**Expected behavior and actual behavior:**
Boost V from the 4 pieces Onslaught set on the Witch Hunter should increase critical Hit Rate by 9%, and Critical damage by 12%.

Currently the boost only increase the latter (see getstat on screen displaying a correct 62 = 50 + 12 %), but the former does not show in getstat, despite being correctly displayed on the paperdoll (second screen showing an increase of 9% only on paperdoll).

Note that the fanatical cleansing tactic is slotted, and correctly taken into account (I dont think it could be related)

**Steps to reproduce the problem:**
Equip WH onslaught 4 pieces (on my char everything saving gloves), get the proc to fire and do a .getstat while buff active.

**Testing Screenshots/Videos/Evidences (always needed):**
Base values are correct without proc buff :
![WH_Onslaught_4pcs_Crit_bug_01](https://user-images.githubusercontent.com/40425011/105043706-280c7080-5a66-11eb-8379-fc06304c6e33.jpg)

After buff, only taking into account the +12% Critical Damage, not the +9% Critical Chance :
![WH_Onslaught_4pcs_Crit_bug_02](https://user-images.githubusercontent.com/40425011/105043747-33f83280-5a66-11eb-97b7-b5da6ece8357.jpg)

Another evidence on a keep lord without other specific buff on me :
![WH_Onslaught_4pcs_Crit_bug](https://user-images.githubusercontent.com/40425011/105043967-76217400-5a66-11eb-930d-7082efbb4c79.jpg)

**Comment by Zumos** [comment]
Under CriticalDamage you see +9 CriticalHitRate.

**Comment by CrimsonFenris** [comment]
> 
> 
> Under CriticalDamage you see +9 CriticalHitRate.

Should the MeleeCritRate display 23 (9 from renown, 5 from stuff and another 9 from the proc), or should it be read MeleeCritRate + CriticalHitRate = 23 ? It's a bit odd to not have it added to all crit rates in the getstats...

Anyway the hitrate line is added while under buff effect, wich could tell the buff is active for all crit rates and not only melee (I guess it affects healing, magic and ranged crit as well, like does the SM tactic), and that's why it takes a separate line ?

Sorry if I overlooked that, will test on targets the effective crit rate (not easy to do since no target have a baseline 0% ctbc...).

**Comment by Zumos** [comment]
> Anyway the hitrate line is added while under buff effect, wich could tell the buff is active for all crit rates and not only melee (I guess it affects healing, magic and ranged crit as well, like does the SM tactic), and that's why it takes a separate line ?

Yes the proc affects all crit rates and that is why it is displayed on a seperate line.

**Comment by grunbag** [comment]
it's just separated stat in get stats, but it correctly increase all crit chances

**Comment by grunbag** [comment]
openeing for @dalen
